coalib/bearlib/abstractions/Linter.py

```diff
@@ -672,9 +688,12 @@ def linter(executable: str,
         ``'corrected'``. By default this value is
         ``coalib.results.RESULT_SEVERITY.NORMAL``. The given value needs to be
         defined inside ``coalib.results.RESULT_SEVERITY``.
-    :param diff_message:
-        The message-string to use for all results if ``output_format`` is
-        ``'corrected'``. By default this value is ``"Inconsistency found."``.
+    :param result_message:
+        The message-string to use for all results. Can be used only together
+        with ``corrected`` or ``regex`` output format. When using
+        ``corrected``, the default value is ``"Inconsistency found."``, while
+        for ``regex`` this static message is disabled and the matched message
+        is used instead.
```

Comment:
maybe I should include "... the message *matched by `output_regex`* is used instead"